.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: Main
+on: [push]
+
+jobs:
+  build:
+    name: Build
+    runs-on: [self-hosted, AWS, Linux]
+    timeout-minutes: 10
+    steps:
+      - uses: Brightspace/third-party-actions@actions/checkout
+      - name: Setup .NET Core
+        uses:  Brightspace/third-party-actions@actions/setup-dotnet
+        with:
+          dotnet-version: 7.0.x

```

Comment:
Most of our other libraries are on v6, but my local machine has v7 installed and everything compiles. Any reason not to upgrade?